roles/os_hardening/templates/etc/audit/auditd.conf.j2

```diff
@@ -2,30 +2,26 @@
 # Generated by Ansible role {{ ansible_role_name }}
 
 log_file = /var/log/audit/audit.log
-log_format = RAW
+log_format = {{ os_auditd_log_format }}
 log_group = root
```

Comment:
Could you add {{ os_auditd_log_group }} to change this value by `adm` for example?

This would be great for log parsers running on non-root user!